### PR TITLE
Add env var example

### PR DESCRIPTION
**Pull request by lucperkins**
Addressing issue #8907

### CODE REVIEW

**Comment by neolit123** [comment]
the change looks good. but please double check if the numbered lists are rendered properly in a netlify preview, as there are markdown bugs at large in Hugo.

having said that the netlify preview seems to be missing here.

**Comment by lucperkins** [comment]
@neolit123 For sure. Everything is fine in this case (I always run locally *and* check Netlify builds).

File: content/en/docs/tasks/inject-data-application/define-environment-variable-container.md [hunk 1]
```diff
@@ -80,6 +118,3 @@ will override any environment variables specified in the container image.
 * See [EnvVarSource](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#envvarsource-v1-core).
 
 {{% /capture %}}
-
-
```
Comment (stewart-yu) [inline content/en/docs/tasks/inject-data-application/define-environment-variable-container.md]: Why delete?

File: content/en/docs/tasks/inject-data-application/define-environment-variable-container.md [hunk 2]
```diff
@@ -36,33 +36,45 @@ Pod:
 
 1. Create a Pod based on the YAML configuration file:
 
-        kubectl create -f https://k8s.io/examples/pods/inject/envars.yaml
+    ```shell
```
Comment (stewart-yu) [inline content/en/docs/tasks/inject-data-application/define-environment-variable-container.md]: Why change? original page looks good https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/

File: content/en/docs/tasks/inject-data-application/define-environment-variable-container.md [hunk 3]
```diff
@@ -36,33 +36,45 @@ Pod:
...
+    kubectl create -f https://k8s.io/examples/pods/inject/envars.yaml
+    ```
 
 1. List the running Pods:
 
-        kubectl get pods -l purpose=demonstrate-envars
+    ```shell
+    kubectl get pods -l purpose=demonstrate-envars
+    ```
```
Comment (stewart-yu) [inline content/en/docs/tasks/inject-data-application/define-environment-variable-container.md]: Same

File: content/en/docs/tasks/inject-data-application/define-environment-variable-container.md [hunk 4]
```diff
@@ -36,33 +36,45 @@ Pod:
...
 
     The output is similar to this:
 
-        NAME            READY     STATUS    RESTARTS   AGE
-        envar-demo      1/1       Running   0          9s
+    ```log
+    NAME            READY     STATUS    RESTARTS   AGE
+    envar-demo      1/1       Running   0          9s
+    ```
```
Comment (stewart-yu) [inline content/en/docs/tasks/inject-data-application/define-environment-variable-container.md]: The same

File: content/en/docs/tasks/inject-data-application/define-environment-variable-container.md [hunk 5]
```diff
@@ -36,33 +36,45 @@ Pod:
...
 
 1. Get a shell to the container running in your Pod:
 
-        kubectl exec -it envar-demo -- /bin/bash
+    ```shell
+    kubectl exec -it envar-demo -- /bin/bash
+    ```
```
Comment (stewart-yu) [inline content/en/docs/tasks/inject-data-application/define-environment-variable-container.md]: The same

File: content/en/docs/tasks/inject-data-application/define-environment-variable-container.md [hunk 6]
```diff
@@ -36,33 +36,45 @@ Pod:
...
 
 1. In your shell, run the `printenv` command to list the environment variables.
 
-        root@envar-demo:/# printenv
+    ```shell
+    root@envar-demo:/# printenv
+    ```
```
Comment (stewart-yu) [inline content/en/docs/tasks/inject-data-application/define-environment-variable-container.md]: The same

File: content/en/docs/tasks/inject-data-application/define-environment-variable-container.md [hunk 7]
```diff
@@ -36,33 +36,45 @@ Pod:
...
-        DEMO_GREETING=Hello from the environment
-        DEMO_FAREWELL=Such a sweet sorrow
+    ```log
+    NODE_VERSION=4.4.2
+    EXAMPLE_SERVICE_PORT_8080_TCP_ADDR=10.3.245.237
+    HOSTNAME=envar-demo
+    ...
+    DEMO_GREETING=Hello from the environment
+    DEMO_FAREWELL=Such a sweet sorrow
+    ```
```
Comment (stewart-yu) [inline content/en/docs/tasks/inject-data-application/define-environment-variable-container.md]: The same

File: content/en/docs/tasks/inject-data-application/define-environment-variable-container.md (hunk 1 above)
Comment (lucperkins) [reply content/en/docs/tasks/inject-data-application/define-environment-variable-container.md]: Why not? These lines are extraneous

File: content/en/docs/tasks/inject-data-application/define-environment-variable-container.md (hunk 7 above)
Comment (lucperkins) [reply content/en/docs/tasks/inject-data-application/define-environment-variable-container.md]: The Black Friday Markdown engine is very finicky about indentation. Four spaces are necessary in this case so we should use no more than that. Furthermore, it’s standard throughout the docs to use backtick-based code blocks and to indicate the appropriate language where possible.

**Pull request merged**